app/graphql/types/base_object.rb

```diff
@@ -9,5 +9,9 @@ class BaseObject < GraphQL::Schema::Object
     edge_type_class(Types::BaseEdge)
     connection_type_class(Types::BaseConnection)
     field_class Types::BaseField
+
+    def current_user
+      context[:current_user]
+    end
```

Comment:
@gigxz , is this correct, or should it be `context[:current_hmis_api_user]` or similar?

Reply:
@eanders Is this fine, or do you want a change here?